js/screens/ChildInfo/ChildInfo.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, ScrollView } from 'react-native';
+import ProfileCard from '../../components/ProfileCard';
+import OutlinedButton from '../../components/OutlinedButton';
+import PropTypes from 'prop-types';
+import styles from './styles';
+import { colors } from '../../config/styles';
+import ChildInfoForm from '../../components/ChildInfoForm';
+
+const Button = ({ pressed }) => (
+  <TouchableOpacity onPress={() => pressed}>
+    <Text style={styles.buttonText}>Change Avatar</Text>
+  </TouchableOpacity>
+);
+
+const ChildInfo = ({
+  child,
+  editChildInfo,
+  showEditInfo,
+  updateChildInfo,
+  updateFormValues,
+}) => {
+  const { info } = child;
+  return (
+    <View style={styles.root}>
+      <ScrollView>
+        <View style={styles.buttonContainer}>
+          <OutlinedButton
+            title={editChildInfo ? 'Done' : 'Edit'}
+            color={colors.teal}
+            pressed={
+              editChildInfo ? () => updateChildInfo() : () => showEditInfo()
+            }
+            titleStyle={{
+              marginTop: -5,
+              marginBottom: -7,
```

Comment:
Please remove the negative margin and try to find an alternative approach.

Reply:
the only approach I could come up with is to not use the designer's font. the font comes with its own built-in padding. the margin removes it to make the button match the designer's vision.